files/templates/additional-scrape-configs.template.yaml

```diff
@@ -362,3 +362,41 @@
     replacement: tenant_cluster
   - target_label: provider
     replacement: {{ .Provider }}
+# app-operator
```

Comment:
Should we not only add those if we are on the host cluster? Also, this reminds me we do not scrape the chart operator on the tenant clusters